tests/versioning_3_test.go

```diff
@@ -1695,33 +1708,12 @@ func respondCompleteWorkflow(
 			},
 		},
 		ForceCreateNewWorkflowTask: false,
-		Deployment:                 tv.Deployment(),
 		VersioningBehavior:         behavior,
-	}
-}
-
-func respondScheduleNexusOperation(
-	tv *testvars.TestVars,
-	behavior enumspb.VersioningBehavior,
-	endpointName string,
-) *workflowservice.RespondWorkflowTaskCompletedRequest {
-	return &workflowservice.RespondWorkflowTaskCompletedRequest{
-		Commands: []*commandpb.Command{
-			{
-				CommandType: enumspb.COMMAND_TYPE_SCHEDULE_NEXUS_OPERATION,
-				Attributes: &commandpb.Command_ScheduleNexusOperationCommandAttributes{
-					ScheduleNexusOperationCommandAttributes: &commandpb.ScheduleNexusOperationCommandAttributes{
-						Endpoint:  endpointName,
-						Service:   tv.Service(),
-						Operation: tv.Operation(),
-						Input:     tv.Any().Payload(),
-					},
-				},
-			},
+		DeploymentOptions: &deploymentpb.WorkerDeploymentOptions{
+			BuildId:              tv.BuildID(),
+			DeploymentName:       tv.DeploymentSeries(),
+			WorkerVersioningMode: enumspb.WORKER_VERSIONING_MODE_VERSIONED,
```

Comment:
do you need a version stamp here too?

Reply:
well, I someone whats to check the SAs after wf completion yes but I didn't have that use and left it out for now because wanted to make sure the request goes through without stamp as well.